plugins/modules/inventory_workflow_manager.py

```diff
@@ -1870,25 +2174,42 @@ def get_site_type(self, site_name):
         """
 
         try:
```

Comment:
Please write this API in dnac.py under site section..

Reply:
updated the code